### ISSUE TITLE
Make https standard

### ISSUE THREAD

**Issue by stefan-balke** [ismir/ismir-home#19]
Can we configure the webspace such that even if I go to http, it's redirected to https by default?

**Comment by ejhumphrey** [comment]
hum, looks like yes. https://support.hostgator.com/articles/ssl-certificates/hostgator-free-ssl

not sure what (if any) are the downsides of a free vs paid SSL. Looks like it's ≈$50 USD for a paid one.

**Comment by Bomme** [comment]
Maybe it can directly be set in the project settings?
https://github.blog/2018-05-01-github-pages-custom-domains-https/#configuring-your-domain

**Comment by hendriks73** [comment]
I'd love to look into it, since not doing it right leads to broken links.
Try opening http://ismir.net/conferences/ (via HTTP) and click on the ISMIR 2020 link. It fails (for the other conferences as well). The same works when using https://ismir.net/conferences/ (via HTTPS).

**Comment by ajaysmurthy** [comment]
Thanks @hendriks73 for the pointers. This issue is now fixed. We used the free SSL provided by Hostgator.

**Comment by hendriks73** [comment]
Did adding a suitable `.htaccess` file, as described [here](https://www.hostgator.com/help/article/i-have-purchased-a-ssl-from-you-now-how-do-i-use-it), solve the issue?

**Comment by ajaysmurthy** [comment]
Modifying .htaccess file using instructions [here](https://www.hostgator.com/help/article/hostgator-free-ssl) solved the issue. 

TL;DR

```
RewriteEngine On
RewriteCond %{HTTPS} !=on
RewriteRule ^ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
```

Additionally, added the following script to fix the Mixed Content warning on your site.
```
RewriteOptions inherit
RewriteEngine on
Header set content-Security-Policy: upgrade-insecure-requests
```
